python/ray/serve/_private/application_state.py

```diff
@@ -388,7 +388,9 @@ def deploy_config(
             )
 
     def _get_live_deployments(self) -> List[str]:
-        return self._deployment_state_manager.get_deployments_in_application(self._name)
+        deps = self._deployment_state_manager.get_deployments_in_application(self._name)
+        prefix = self._name + "_"
+        return [deployment[len(prefix) :] for deployment in deps]
```

Comment:
why do we need to do the prefix parsing here instead of having `get_deployments_in_application` return the raw deployment name or the new DeploymentID?